README.md

```diff
@@ -1,2 +1,28 @@
: "http://localhost:8050/openmrs/ws/rest/v1/person/a751f0da-39ab-45a3-db52-21706f5ed201/name/70dfed83-a323-4832-d73e-fc667b049773" }, { "rel": "full", "uri": "http://localhost:8050/openmrs/ws/rest/v1/person/a751f0da-39ab-45a3-db52-21706f5ed201/name/70dfed83-a323-4832-d73e-fc667b049773?v=full" } ], "resourceVersion": "1.8" } ], "addresses": [ ], "attributes": [ { "display": "General", "uuid": "396db6e6-0f01-4279-8646-76ac3c862595", "value": { "uuid": "c1fc20ab-3f10-11e4-adec-0800271c1b75", "display": "General", "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/concept/c1fc20ab-3f10-11e4-adec-0800271c1b75" } ] }, "attributeType": { "uuid": "c1f455e7-3f10-11e4-adec-0800271c1b75", "display": "class", "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/personattributetype/c1f455e7-3f10-11e4-adec-0800271c1b75" } ] }, "voided": false, "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/person/a751f0da-39ab-45a3-db52-21706f5ed201/attribute/396db6e6-0f01-4279-8646-76ac3c862595" }, { "rel": "full", "uri": "http://localhost:8050/openmrs/ws/rest/v1/person/a751f0da-39ab-45a3-db52-21706f5ed201/attribute/396db6e6-0f01-4279-8646-76ac3c862595?v=full" } ], "resourceVersion": "1.8" }, { "display": "landHolding = 2", "uuid": "de08e0bf-b6b1-4226-9a10-c1593b67fd65", "value": 2, "attributeType": { "uuid": "3dfdc176-17fd-42b1-b5be-c7e25b78b602", "display": "landHolding", "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/personattributetype/3dfdc176-17fd-42b1-b5be-c7e25b78b602" } ] }, "voided": false, "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/person/a751f0da-39ab-45a3-db52-21706f5ed201/attribute/de08e0bf-b6b1-4226-9a10-c1593b67fd65" }, { "rel": "full", "uri": "http://localhost:8050/openmrs/ws/rest/v1/person/a751f0da-39ab-45a3-db52-21706f5ed201/attribute/de08e0bf-b6b1-4226-9a10-c1593b67fd65?v=full" } ], "resourceVersion": "1.8" } ], "voided": false, "auditInfo": { "creator": { "uuid": "c1c21e11-3f10-11e4-adec-0800271c1b75", "display": "superman", "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/user/c1c21e11-3f10-11e4-adec-0800271c1b75" } ] }, "dateCreated": "2017-06-07T16:05:31.000+0530", "changedBy": null, "dateChanged": null }, "birthtime": null, "deathdateEstimated": false, "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/person/a751f0da-39ab-45a3-db52-21706f5ed201" } ], "resourceVersion": "1.11" }, "voided": false, "auditInfo": { "creator": { "uuid": "c1c21e11-3f10-11e4-adec-0800271c1b75", "display": "superman", "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/user/c1c21e11-3f10-11e4-adec-0800271c1b75" } ] }, "dateCreated": "2017-06-07T16:05:31.000+0530", "changedBy": null, "dateChanged": null }, "links": [ { "rel": "self", "uri": "http://localhost:8050/openmrs/ws/rest/v1/patient/a751f0da-39ab-45a3-db52-21706f5ed201" } ], "resourceVersion": "1.8" }
```

Comment:
This should be a json file. Not in main readme.md. Only put reference to this file in Readme.md.

Reply:
Added a reference to json file